src/main/features/claim/views/eligibility/over-18-defendant.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, radioGroup, saveAndContinueButton %}
+
+{% set title = t('Over 18 | Money claim') %}
```

Comment:
Shall we rename title to indicate we are asking about defendant age? Just asking, we have no consistent approach for page titles.